### ISSUE TITLE
Allow any configuration file for the test suite

### ISSUE THREAD

**Issue by JulienPeloton** [astrolabsoftware/fink-broker#148]
**Describe the issue**
The current test suite uses the configuration file `conf/fink.conf.travis`, and the user cannot switch to a different configuration file.

**Wanted behaviour**
The script should have a default configuration file (`conf/fink.conf.travis`), but should also let the user overwrite it:

```bash
# Default will use conf/fink.conf.travis
fink_test 

# Will use my-conf.conf
fink_test -c my-conf.conf
```

**Comment by cAbhi15** [comment]
@JulienPeloton can I fix this issue? or is it being fixed as a part of a larger patch?

**Comment by JulienPeloton** [comment]
@cAbhi15 feel free to fix it! I'm not on it currently. Thanks.

**Comment by JulienPeloton** [comment]
Fixed in #163. Thanks @cAbhi15 !
